test/foundry/unit/StakingRewardsV2/StakingV2Checkpointing.t.sol

```diff
@@ -957,7 +930,7 @@ contract StakingV2CheckpointingTests is DefaultStakingV2Setup {
                                 Helpers
     //////////////////////////////////////////////////////////////*/
 
-    function xCooldownPeriods(uint256 numCooldowns) public view returns (uint256) {
-        return 1 + (numCooldowns * stakingRewardsV2.cooldownPeriod());
+    function xCooldownPeriods(uint256 numCooldowns) public pure returns (uint256) {
```

Comment:
**Thought:** Can this function be deleted now?

Reply:
Still used in the `Before_Block` and `After_Block`, not sure how much these tests make sense right now, i propose to simply rename it `xPeriods` as Cooldown does not mean much anymore, this way we can keep the existing tests.

Reply:
Okay cool makes sense.